pkgs/development/haskell-modules/configuration-ghc-9.0.x.nix

```diff
@@ -133,4 +133,9 @@ self: super: {
 
   # 2022-05-31: weeder 2.3.0 requires GHC 9.2
   weeder = doDistribute self.weeder_2_3_1;
+
+  # 2022-09-01:
+  # Restrictive upper bound on base.
+  # Remove once version 1.* is released
+  monad-bayes = doJailbreak super.monad-bayes;
```

Comment:
Thanks for this PR!

If this type of change affects our current main version of GHC (and all future versions), then we generally just put it in `configuration-common.nix` instead of the compiler-specific files (like `configuration-ghc-9.0.x.nix` and `configuration-ghc-9.2.x.nix`.)

`doJailbreak` is particularly harmless as well, since it is unlikely to break the build for ghc-8s.

Would you be able to make this change?

Reply:
Sure :) applied change and rebased.